RingPublishingTracking/src/main/java/com/ringpublishing/tracking/internal/decorator/UserSsoDataDecorator.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ *  Created by Grzegorz Małopolski on 10/6/21, 11:28 AM
+ * Copyright © 2021 Ringier Axel Springer Tech. All rights reserved.
+ *
+ */
+
+package com.ringpublishing.tracking.internal.decorator
+
+import android.util.Base64
+import com.google.gson.Gson
+import com.ringpublishing.tracking.data.Event
+import com.ringpublishing.tracking.internal.delegate.ConfigurationManager
+import com.ringpublishing.tracking.internal.log.Logger
+import java.io.UnsupportedEncodingException
+
+internal class UserSsoDataDecorator(private val configurationManager: ConfigurationManager, private val gson: Gson) : BaseDecorator()
+{
+	private val userId get() = configurationManager.getUserData().userId
+	private val ssoName get() = configurationManager.getUserData().ssoName ?: ""
```

Comment:
configuracja sie nie zmienia to po co za kazdym razme jest sprawdzany czy nie jest to null ?

troche nie kumam dlczego uzywasz w taki sposob get()